autocrop/cli.py

```diff
@@ -189,6 +194,18 @@ def confirmation(question):
         print(notification_str)
 
 
+def chk_extension(extension):
+    """Check if the extension passed is valid or not."""
+    error = "Invalid image extension"
+    extension = str(extension).lower()
+    if not extension.startswith("."):
+        extension = f".{extension}"
+    if extension in COMBINED_FILETYPES:
+        return extension.lower().replace(".", "")
```

Comment:
I moved `lower` and `replace('.', '')` here. Since we're making sure the extension is correct, might as well standardize it too.